artist-api/src/main/kotlin/com/uber/artist/api/KotlinTypeNames.kt

```diff
@@ -57,6 +57,7 @@ class KotlinTypeNames {
   class Java {
     companion object {
       val Map: ClassName = java.util.Map::class.asClassName()
+      val Object: ClassName = java.lang.Object::class.asClassName()
```

Comment:
We should be using `Any` or `Unit` for Kotlin right?

Reply:
Can we not make it idiomatic kotlin here?

Reply:
done